package-lock.json

```diff
@@ -8,11 +8,11 @@
       "integrity": "sha512-ciiioYMLdo16ShmfHBXJBOFm3xPC4AuwO4xeRpFeHz7WK9PYsWCmigagG2XyzZpubK4a3qNKoUBDhbzHfa50LQ==",
       "dev": true,
       "requires": {
-        "acorn": "5.7.1",
```

Comment:
There are a lot of changes in `package-lock.json` but nothing else to indicate something package-wise has changed. Are you using a newer version of npm than before?

Reply:
Reverted